### DISCUSSION TITLE
Flashing Matek F405 WMN not possible with win11

### DISCUSSION THREAD

**Discussion by ghost** [iNavFlight/inav#8703] [Q&A]
Trying to flash a Matek F405 WMN with INAV. Flashing is not possible. 
Also tried to set USB driver for STM, which is not possible because not visible with zadig (STM driver installed direcly from STM)
F405TE Driver is seen, installed (with problems) but even then not working
Does anyone had same issues?

Flashing does not work, 
Reaching the setup page was only sporadically possible

## Steps to Reproduce
Could be reproduced by INAV 5.1 INAV 6.0 FP1 and FP2
1.
2.
3.
4.

## Additional context

---

- FC Board name and vendor:
- INAV version string:

**Comment by kasatka60** [comment]
Try Disable Driver Signature Enforcement in windows 11.
zadig: options->Check the box next to "All list Devices"

**Reply by ghost** [reply]
Thanks for your reply, but unfortunately it doesn't work
I setted up the same procedure with a PC Win 10 and it worked perfectly
